README.md

```diff
@@ -1,4 +1,4 @@
-# Shopware Development Documentation
+# Shopware Development Documentation - Best for Developers
```

Comment:
why do want to add this?

Reply:
It's most probably hacktoberfest spam.

Reply:
I already thought so, too. Will close this